README.md

```diff
@@ -22,7 +22,12 @@ If you want to use the specified shared configs:
 
 ```bash
 # use `eslint-config-semistandard` shared config
+# npm 6.x
 npm init @eslint/config --config semistandard
+
+# ⚠️ npm 7+, extra double-dash is needed:
+npm init @eslint/config -- --config semistandard
```

Comment:
This looks good. Maybe we could also just add the `--` to the remaining examples below?